### ISSUE TITLE
Base routing not working

### ISSUE THREAD

**Issue by baio** [visionmedia/page.js#228]
url - .../#!/user/edit

router 

```
 page('/user/*', function() {
  console.log("user/*");
 //this NOT invoked
 });

 page('/user/edit', function() {
  console.log("user/edit");
 //this invoked
 });

 page({hashbang : true});
```

The base path callback is not invoked!

page.js "version": "1.6.1",
Chrome

**Comment by JChapman202** [comment]
This issue is not limited to hashbang: true.  It exists for both.  Additionally you can see this issue exhibited within the profile example included with page.js.

In that example you'll see the following routes:

```
page('/user/*', displayIndexAfter(5000));
// you dont need to use two,
// but this demonstrates how
// you can "filter" requests
// then move on to the next callback
// to separate concerns
page('/user/:name', load);
page('/user/:name', show)
```

However, when you run the example the 5 second timer never fires.  The page stays on that view forever since the `/user/*` is never matched by page.js.

**Comment by JChapman202** [comment]
At least related to the profile example above, this appears to be a documentation and example issue.  The underlying library [path-to-regexp](https://github.com/pillarjs/path-to-regexp) was changed to no longer support the /* wildcard routing.  Instead it relies on using (.*) see: [path-to-regexp#compatability-with-express--4x](https://github.com/pillarjs/path-to-regexp#compatibility-with-express--4x).  

If you change the example to use:

```
page('/user/(.*)', displayIndexAfter(5000));
```

it will work fine again.  To me the two options here would either to do what the base * handler does and replace a * with (.*), or alternatively document this feature to work with (.*) only.

These are both relatively simple changes that I could make.  Let me know your thoughts?

**Comment by laicosly** [comment]
the behavior I'm seeing with .base makes no fucking sense...

**Comment by woodlandhunter** [comment]
@shuvalov-anton Recommend closing. @baoi's original issue is fixed in the 1.6.4 release

@laicosly Please open a new issue describing your problems with `page.base()`. The original issue here is not about `page.base()`
